merge_requests.go

```diff
@@ -123,9 +123,10 @@ type MergeRequest struct {
 		HeadSha  string `json:"head_sha"`
 		StartSha string `json:"start_sha"`
 	} `json:"diff_refs"`
-	DivergedCommitsCount int  `json:"diverged_commits_count"`
-	RebaseInProgress     bool `json:"rebase_in_progress"`
-	ApprovalsBeforeMerge int  `json:"approvals_before_merge"`
+	DivergedCommitsCount int     `json:"diverged_commits_count"`
+	RebaseInProgress     bool    `json:"rebase_in_progress"`
+	ApprovalsBeforeMerge int     `json:"approvals_before_merge"`
+	MergeError           *string `json:"merge_error"`
```

Comment:
I think this field should be blow `MergeStatus` in order to keep the same ordering as the docs (which I try to use as guideline). And I lso think this shouldn't be a pointer to a string, but just a string...

Reply:
Thanks for the comment. I've adjusted the PR.